application/views/admin/survey/activateSurvey_view.php

```diff
@@ -183,6 +187,15 @@ function alertPrivacy()
         <p class='col-sm-7 col-sm-offset-2'>
             <?php eT("Please note that once responses have collected with this survey and you want to add or remove groups/questions or change one of the settings above, you will need to deactivate this survey, which will move all data that has already been entered into a separate archived table."); ?><br /><br />
         </p>
+        <?php if($oSurvey->getIsDateExpired()):?>
+        <div class="row">
+            <div class='col-sm-7 col-sm-offset-2'>
+
+                <div class="alert alert-danger"><?= Yii::t('app', 'NB! The survey you are activating is currently expired! The survey will not be available for participants.')?><div>
```

Comment:
What's 'NB' ?  Also should be a note not a warning - it is not like you cannot remove the expiration date after activation.
Also the hint should be helpful, not keep the user from activating a survey. So rather like:

"Note: This survey has a past expiration date configured and is currently not available to participants. Please remember to update/remove the expiration date in the survey settings after activation."

Reply:
Being someone who has activated a number of surveys while the expiry date was set and passed, I would say that treating this as a warning is adequate. 

When one is activating a survey - what is the expected behaviour? The expected behaviour is that after activating the survey will be active and ready to accept responses. If you have an expiry date set in past you will not reach the expected outcome. Thus - a warning, not a note. 

But of course it's juts my opinion. If you really feel it should be less visible, I will change it of course, but I personally will clearly regard this as an warning not a mere note. 

ps.
NB is Nota bene :)
https://en.wikipedia.org/wiki/Nota_bene

Reply:
@TonisOrmisson you set it as alert-danger, can be moved to alert-warning ;)

Reply:
have you seen alert-warning? :)

Reply:
@Shnoulle I can live with alert-warning :) thanks

Reply:
Not an issue ;) personally : danger is OK for me, warning more for «active but not started : don't send the url now, but in some day :)». Just point of view, not major.